packages/build-info/src/frameworks/waku.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseFramework, Category, Framework } from './framework.js'
+
+export class Waku extends BaseFramework implements Framework {
+  readonly id = 'waku'
+  name = 'Waku'
+  npmDependencies = ['waku']
+  configFiles = ['waku.config.ts', 'waku.config.ts']
+  category = Category.SSG
+
+  dev = {
+    command: 'waku dev',
+    port: 3000,
+  }
+
+  build = {
+    command: 'waku build',
+    directory: 'dist',
```

Comment:
Looks like the [publish directory](https://docs.netlify.com/build/configure-builds/overview/#definitions) is `dist/public`

Reply:
yes, sorry it's the same confusion as the other PR 🙂👍